modules/i3s/src/types.ts

```diff
@@ -157,7 +157,7 @@ export type BoundingVolumes = {
 export type Obb = {
   center: number[] | Vector3;
   halfSize: number[] | Vector3;
-  quaternion: Quaternion;
+  quaternion: number[] | Quaternion;
```

Comment:
We have a `NumericArray` type in math.gl that also covers typed arrays.

Reply:
There is not this type in the version we use in I3S module